cmd/runtimetest/main.go

```diff
@@ -576,6 +576,25 @@ func validateMaskedPaths(spec *rspec.Spec) error {
 	return nil
 }
 
+func validateSeccomp(spec *rspec.Spec) error {
+	if spec.Linux == nil || spec.Linux.Seccomp == nil {
+		return nil
+	}
+	for _, sys := range spec.Linux.Seccomp.Syscalls {
+		if sys.Action == "SCMP_ACT_ERRON" {
+			for _, name := range sys.Names {
+				if name == "getcwd" {
+					_, err := os.Getwd()
+					if err == nil {
+						logrus.Warnf("Syscall action %v can not be properly applied in the runtime", sys.Action)
```

Comment:
This is a non-TAP warning that will be [gobbled by `RuntimeInsideValidate`][1].  I think you want something like:

```go
t.Skip(1, fmt.Sprintf("%s syscall returns errno", sys.Name))
if err == nil {
  t.Diagnostic("did not return an error")
} else {
  t.Diagnosticf("returned %d (%s)", err.WhateverOtherErrno(), err.Error())
}
```

[1]: https://github.com/opencontainers/runtime-tools/blob/c3755c1175a1baad7230dfb03751e3a4b7182f32/validation/util/test.go#L131-L142

Reply:
I tried to modify it, what do you think?